### DISCUSSION TITLE
use different nicknames for different websites?

### DISCUSSION THREAD

**Discussion by comsyspro** [LiveHelperChat/livehelperchat#1841] [General]
i have 2 websites and created 2 departments. i implemented the embbed widget code for the 2 websites each with seperate assigned department. now is it possible to have different nicknames for each departments when using only one (the same) admin  user? so if there is a chat starting on department 1 there the nickname should be nick1 and for department 2 the nickname should be nick2, all with only one user account.

**Comment by remdex** [comment]
Hi, at the moment there is no such feature. You would need to use - two browsers I'm afraid now if you wanted it now. There is feature join as, but it requires that chat were accepted by other operator in any case :)
It's possible to develop with sponsorship if interested. But not so fast 😀
